std/format/package.d

```diff
@@ -144,6 +144,59 @@ recommended to assign (lower- and uppercase) letters.
 Note: The $(I Parameters) of a $(I CompoundIndicator) are currently
 limited to a $(B '-') flag.
 
+$(SECTION4 Format Indicator)
```

Comment:
[The `printf` manual uses the term "Conversion specifier"](https://man7.org/linux/man-pages/man3/printf.3.html):
> Each _conversion specification_ is introduced by the character %, and ends with a _conversion specifier_.

Reply:
When I started to rewrite the docs here, I encountered different namings for the same thing at various places. For example have a look at [printf in C++](https://www.cplusplus.com/reference/cstdio/printf/). It uses the word "specifier" for the whole thing as well as the letter at the end. In the former docs of `std.format` it wasn't used consistent either.

So I tried to unify the wording throughout the whole module, which resulted in a rewrite of the grammar (see PR #7927). IMHO this wording is consistent in itself and changing a single term would make other changes necessary. For example I'm using "format specifier" for the whole thing. If we use "conversion specifier" here, this needs to be changed too, to avoid confusion.

Another thing, that comes to my mind: Although `printf` was originally the model for this package, `std.format` has already diverged over the time (grouping is done with comma, not with a tick; there is no need to tell about `long` values in D, `%s` was added, and so on) and in my opinion we should not cling too strong to `printf` - it has some drawbacks (e.g. `%p` is unsafe and luckily was never added to Phobos). IMHO D can do better. :-)